python_modules/libraries/dagster-components/dagster_components/core/schema/objects.py

```diff
@@ -155,6 +155,12 @@ class AssetSpecResolutionSpec(ResolutionSpec):
     automation_condition: Optional[AutomationCondition]
 
 
+AssetSpecSequenceField: TypeAlias = Annotated[
```

Comment:
@alangenfeld note when you work on the more generic engine for iterating over typing primitives this should be deleted